scripts/yast2

```diff
@@ -35,6 +35,10 @@ export QT_IM_MODULE=xim GTK_IM_MODULE=xim
 # used correctly (see bnc#866692 for details).
 export PX_MODULE_PATH=""
 
+# Force using Perl readline library instead of the broken default Gnu variant
+# to avoid ugly error message printed in command line mode (bsc#966042)
+export PERL_RL=Perl
```

Comment:
Thanks for the catch! But then your proposed ideas did not really improve the situation, so I tried to look into the underlying package, and found that updating to the latest upstream version fixes it: https://bugzilla.suse.com/show_bug.cgi?id=966042#c7